packages/@aws-cdk/aws-apigatewayv2/package.json

```diff
@@ -103,13 +103,20 @@
   },
   "awslint": {
     "exclude": [
+      "props-physical-name:@aws-cdk/aws-apigatewayv2.ApiMappingProps",
+      "construct-interface-extends-iconstruct:@aws-cdk/aws-apigatewayv2.IHttpApi",
+      "construct-interface-extends-iconstruct:@aws-cdk/aws-apigatewayv2.IWebSocketApi",
+      "resource-interface-extends-resource:@aws-cdk/aws-apigatewayv2.IHttpApi",
+      "resource-interface-extends-resource:@aws-cdk/aws-apigatewayv2.IWebSocketApi",
```

Comment:
That's odd. Is this because these interfaces don't directly inherit `IConstruct` and `IResource`? 
Can you check if something is amiss in this code? Otherwise, I'll have to look into the linter to see if there's a bug there.

Reply:
Yes I am confused with this as well. Here the reference to `Resource` comes from the base `ApiBase` class and nowhere around `IApi` or `IHttpApi`. Not sure how to address this or maybe a linter issue indeed.

Reply:
I think the problem is that `IApi` does not extend `IResource`. Can you see if that fixes it?

Reply:
Yep that does it. Thanks!